src/components/Dashboard.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+	<DashboardWidget :items="notes"
+		:loading="loading"
+		:show-more-text="t('notes', 'notes')"
+		:show-more-url="showMoreUrl"
+	>
+		<template v-slot:default="{ item }">
+			<DashboardWidgetItem
+				:target-url="getItemTargetUrl(item)"
+				:main-text="item.title"
+				:sub-text="subtext(item)"
+				:item="item"
+			>
+				<template v-slot:avatar>
+					<div
+						class="note-item"
+						:class="{ 'note-item-favorite': item.favorite }"
+					/>
+				</template>
+			</DashboardWidgetItem>
+		</template>
+		<template v-slot:empty-content>
+			<EmptyContent icon="icon-notes">
+				<template #desc>
+					<p class="notes-empty-content-label">
+						{{ t('notes', 'No notes found') }}
```

Comment:
Slightly better wording would be "No notes yet" since that communicates the proper state, instead of sounding a bit like they were lost. ;)